scripts/transform-year-organizations.ts

```
@@ -230,6 +231,37 @@ async function main() {
+    }
+    for (const tech of (raw.tech_tags || [])) {
+      const stdTech = getStandardTechName(tech);
+      if (!knownTechs.has(stdTech.toLowerCase())) {
+        newTechsFound.add(stdTech);
+      }
+    }
+  }
+
+  if (unmappedOrgsFound.size > 0) {
+    console.warn(`[WARN] Found ${unmappedOrgsFound.size} new/unmapped Organizations. Consider adding them to ORG_ALIASES in normalize-data.ts:`);
+    Array.from(unmappedOrgsFound).forEach(o => console.warn(`   - "${o}"`));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)